edgelet/iotedge-proxy/src/api.rs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+use failure::{Compat, Fail};
+use futures::future::FutureResult;
+use futures::{future, Future, IntoFuture};
+use hyper::service::{NewService, Service};
+use hyper::{header, Body, Method, Request, Response, StatusCode};
+use log::{debug, info};
+
+use crate::{logging, Error};
+
+#[derive(Clone)]
+pub struct ApiService;
+
+impl ApiService {
+    pub fn new() -> Self {
+        ApiService
+    }
+
+    fn handle(&self, req: &Request<Body>) -> Result<Response<Body>, Error> {
+        match (req.method(), req.uri().path()) {
+            (&Method::GET, "/health") => Ok(Response::new(Body::empty())),
```

Comment:
Intended for k8s to make liveness probe. once it cannot get 200 response - it'll re-create pod. 
But good idea. Maybe we should ping `iotedged` instead? k8s itself will not be able to make a liveness probe because `iotedged` exposes only https endpoints and k8s doesn't have a valid cert.

@darobs

Reply:
For liveness, I'm inclined to think that being able to receive a request and respond is adequate, until we know any better. It would be nice if we could check on the module container's status, but I don't think we have a consistent liveness method for a module.

Talking to iotedged seems more like a "readiness" check.